config/initializers/time.rb

```diff
@@ -1,7 +1,7 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    getlocal.strftime("%m-%d-%Y")
```

Comment:
No worries.  I changed it because @mebates wanted different formatting, but either way is fine by me.